web/packages/teleport/src/components/Wizard/flow.tsx

```diff
@@ -19,6 +19,7 @@
 export type BaseView<T> = T & {
   hide?: boolean;
   index?: number;
+  displayIndex?: number;
```

Comment:
Could you add JSDoc for `displayIndex` to explain what it's used for and test cases for the new behavior?

Reply:
For sure.